### PR TITLE
EMP-196: Results logic

### PR DESCRIPTION
**Pull request by sarahyjja**

### CODE REVIEW

File: server/data/eqSearchApiClient.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import RestClient from './restClient'
+
+type searchResult = {
```
Comment (muyinatech) [inline server/data/eqSearchApiClient.ts:3]: We can rename this type to **Result**

File: server/data/restClient.ts [hunk 2]
```diff
@@ -82,7 +82,7 @@ export default class RestClient {
     method: 'patch' | 'post' | 'put',
     { path, query = {}, headers = {}, responseType = '', data = {}, raw = false, retry = false }: RequestWithBody,
   ): Promise<Response> {
-    logger.info(`${this.name} ${method.toUpperCase()}: ${path}`)
+    logger.info(`RequestWithBody calling: ${this.name} ${method.toUpperCase()}: ${path}`)
```
Comment (muyinatech) [inline server/data/restClient.ts:85]: We can revert the changes in this file

File: server/routes/index.ts [hunk 3]
```diff
@@ -16,6 +21,17 @@ export default function routes(service: Services): Router {
     res.render('pages/searchEform')
   })
 
+  post('/search-eform', async (req, res, next) => {
+    const restClient = new RestClient('Mirror Gateway API Client', config.apis.eqSearchApi, 'no_auth')
```
Comment (muyinatech) [inline server/routes/index.ts:25]: Replace 'Mirror Gateway API Client' with 'EQ Search API Client'

**Review by muyinatech — APPROVED** [review]
Minor suggested changes

File: server/data/eqSearchApiClient.ts (hunk 1 above)
Comment (psoleckimoj) [reply server/data/eqSearchApiClient.ts:3]: Seeing as everything else here is prefixed with `search` or `eqSearch` I think `Result` doesn't follow. Id disagree and say the current type name is fine. Although we could then argue that line 17 the response interface should be `searchResults: searchResult[]` 🙂  So I'm happy as long as everything is consistent.

File: docker-compose.yml [hunk 4]
```diff
@@ -23,3 +23,4 @@ services:
 
 networks:
   laa:
+
```
Comment (psoleckimoj) [inline docker-compose.yml:26]: Missing trailing newline

**Review by psoleckimoj — APPROVED** [review]
LGTM!

File: server/data/eqSearchApiClient.ts (hunk 1 above)
Comment (sarahyjja) [reply server/data/eqSearchApiClient.ts:3]: Thanks for your reviews @muyinatech @psoleckimoj, I think I will then keep it as it is as I found it it less confusing, but happy to come back on it in the future to make it consistent with the whole project if necessary :)
